monitoring/osd-understanding-the-monitoring-stack.adoc

```diff
@@ -0,0 +1,27 @@
+[id="understanding-the-monitoring-stack"]
```

Comment:
There is no content index at the beginning of the page, for detail refer
https://docs.openshift.com/container-platform/4.7/monitoring/understanding-the-monitoring-stack.html

Reply:
This has been fixed by another PR. It should display when I commit the rest of these content changes.